### ISSUE TITLE
Launch darkly version 3+ crashes older browsers

### ISSUE THREAD

**Issue by gregrealeyes** [launchdarkly/react-client-sdk#182]
**Describe the bug**
Launch darkly version 3+ crashes chrome browser's version 79 or lower.  I noticed this when trying to run a web app on Samsung Tizen TV because they use older browsers. Then I was able to reproduce in an older version of Chrome using BrowserStack

**To reproduce**
Load an app on chrome browser 79 or lower (I used browserstack)

**Expected behavior**
I expect it to load properly on older browsers

**Logs**
<img width="717" alt="Screen Shot 2023-02-01 at 2 07 07 PM" src="https://user-images.githubusercontent.com/41081503/216384995-41855f8e-0556-44e6-b8a8-6005db85ab1f.png">

**SDK version**
3.1.0

**Language version, developer tools**
Node v16.8.0
npm 7.21.0

**OS/platform**
chrome 79
macOS Monterey 12.6.3

**Comment by yusinto** [comment]
The React SDK v3.0 and up uses optional chaining which is supported by modern browsers. It looks like you bundle your react app using webpack. The issue is webpack 4 uses a version of [acorn](https://github.com/webpack/webpack/issues/10227#issuecomment-706752571) which does not transpile the optional chaining operator. The solution as you can see in the react async provider [example](https://github.com/launchdarkly/react-client-sdk/blob/c840be20ab8d370864eab4c6bb2062918efaa891/examples/async-provider/package.json#L63) is to resolve the acorn package version that does support it. To do this, in your package.json file:

```json
  "resolutions": {
    "acorn": "npm:acorn-with-stage3"
  }
```

Webpack 5 supports optional chaining out of the box so we recommend you upgrade to that if possible. If you use other bundlers, make sure to enable/include plugins for optional chaining for your bundler.

**Comment by yauheniivashkevich** [comment]
I have the same issue when run `jest test`
adding resolutions don't help to fix that

**Comment by yusinto** [comment]
This should be fixed in [3.0.2](https://github.com/launchdarkly/react-client-sdk/releases/tag/3.0.2). We have removed the optional chaining operator so please update to that version if you have issues with the `?` operator.
